src/disposable-modal.js

```diff
@@ -3,19 +3,26 @@ import ReactDOM from "react-dom";
 import Cancelable from "react-disposable-decorator";
 import mountComponent from "disposable-component";
 
+@Cancelable
```

Comment:
It does not need to be decorated because of line https://github.com/CanopyTax/react-disposable-modal/pull/1/files#diff-ce8028a752a8c66bf4f375d3076e6990R29

Reply:
Ahh, ok, I wasn't sure. I'll remove it.